CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## [1.5.0]
+
+`2021-05-13`
```

Comment:
Placeholder would be better here.

https://github.com/iTwin/iTwinUI-react/blob/4e5c2f174e8bb2f661d04c979ffbd814e6268a13/CONTRIBUTING.md#L128

Reply:
Then contributing should be modified to just mention the placeholder... Everyone would waste less time updating this.

Reply:
What I mean is that as an external contributor, there is no way I can predict when and in which version this will actually be published, meaning that you guys will always have to validate whatever is in the CHANGELOG, so, the Contrib could be reworded to take that into account. The less doubt there is, the simpler it is to contribute, something like:
``` The `CHANGELOG.md` file must be updated for any new components or changes that you add. Add a placeholder version and date (on the top) and iTwinUI-React team will handle the actual date and version when publishing the package.```
Which tells me that I dont have to worry at all about these 2 informations, but I must put it there, and it will be handled correctly. :)

On top of that, it could be interresting to ask a format of the changelog, that you guys will be able to adapt for consistency and verbosity (just like you did, but instead of me trying to figure what to write right, I could have added only that I added `Header`, `HeaderButton`... components and let you guys figure how you want to surface it.